src/sources/http_server.rs

```diff
@@ -484,6 +510,16 @@ impl HttpSource for SimpleHttpSource {
                         SimpleHttpConfig::NAME,
                         now,
                     );
+
+                    source_ip.map(|addr| {
+                        self.log_namespace.insert_source_metadata(
+                            SimpleHttpConfig::NAME,
+                            log,
+                            self.host_key.path.as_ref().map(LegacyKey::Overwrite),
+                            path!("host"),
+                            addr.to_string(),
```

Comment:
I believe this will include the port number (like 10.12.23.32:12739), is that what you expected?

Reply:
If so, could we update the docs to say they that the value will include the port and not just the IP address?

Reply:
(this also highlights a lack of tests in this PR 😄 Could we add one?)

Reply:
Hey @jszwedko, yes I am aware that this would also include the port number, but I wasn't convinced that we should have it, or exclude it. What do you suggest? If yes, how can this be helpful in observability setup?

Reply:
I feel, we should only have host IP, according to the requirements discussed this far.

Reply:
I agree, I believe that is the precedent we have for our sources (to only include IP address for `host`).